src/screens/MainApp/DiagnoseRequest/ImageReview/ConfirmButton/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Text, Button } from '~/components'
+
+const Error = ({ errorState }) => (
+  errorState &&
+    <Text
+      colorVariant='secondary'
+      style={{ alignSelf: 'center' }}
+    >Selecciona al menos 3 imagenes para continuar!
```

Comment:
I don't personally like this syntax very much:
```javascript
<Component
  prop1={value1}
  prop2={value2}
>Content
</Component>
```
I think it's clearer this way:
```javascript
<Component
  prop1={value1}
  prop2={value2}
>
  Content
</Component>
```

(Just an opinion)